src-electron/db/zap-schema.sql

```diff
@@ -281,6 +281,8 @@ CREATE TABLE IF NOT EXISTS "ATTRIBUTE" (
   "INTRODUCED_IN_REF" integer,
   "REMOVED_IN_REF" integer,
   "API_MATURITY" text,
+  "CHANGE_COMITTED" integer,
```

Comment:
We generally call boolean columns `IS_SOMETHING`. Like `IS_NULLABLE` and such. Should we call this `IS_CHANGE_COMITTED`?